tf_encrypted/convert/register.py

```diff
@@ -50,6 +51,7 @@ def registry():
       'Identity': _identity,
       "GatherV2": _gather,
       "dense": _keras_dense,
+      "batch_normalization_v1": _keras_batchnorm,
```

Comment:
will this work if the user supplies the V2 version of keras's batchnorm?

Reply:
it's okay if it won't, we can leave that for another PR, just curious!

Reply:
I'm not sure yet, trying to understand how to call V2 - in my tf version (1.13.1) batchnorm layer always runs V1, and V2 is for tensorflow 2.0
```
if tf2.enabled():
  BatchNormalization = BatchNormalizationV2
else:
  BatchNormalization = BatchNormalizationV1
```

But perhaps newer tf versions do call V2?

Reply:
ah, that would make sense.  it looked like at one point during 1.xx they added a V2 of batchnorm, but I wasn't sure if that was the same difference that I was seeing here.  thanks!